messaging/pom.xml

```diff
@@ -28,6 +28,13 @@
 			<artifactId>org.eclipse.paho.client.mqttv3</artifactId>
 			<version>1.2.5</version>
 		</dependency>
+		<dependency>
+			<groupId>org.apache.kafka</groupId>
+			<artifactId>kafka-clients</artifactId>
+			<version>3.6.0</version>
+			<scope>provided</scope>
+			<optional>true</optional>
```

Comment:
Why is it provided and optional? Do you assume an application server?

Reply:
> Why is it provided and optional? Do you assume an application server?

I want to make it optional but when looking at other optional dependencies we have, I copy-pasted the "provided" as well. I probably shouldn't